Source/WebCore/inspector/InspectorStyleSheet.cpp

```diff
@@ -119,6 +119,8 @@ static RuleFlatteningStrategy flatteningStrategyForStyleRuleType(StyleRuleType s
     case StyleRuleType::FontFeatureValues:
     case StyleRuleType::FontFeatureValuesBlock:
     case StyleRuleType::FontPaletteValues:
+    // FIXME: probably should implement this?
+    case StyleRuleType::PositionTry:
```

Comment:
Can you check if this works and remove the FIXME as appropriate? If it doesn't then please create a bug for it. If you don't know, I'd rather not include it.

Reply:
https://bugs.webkit.org/show_bug.cgi?id=284176